src/main/java/net/poweredbyhate/wildtp/WildTPListener.java

```diff
@@ -1,9 +1,6 @@
 package net.poweredbyhate.wildtp;
 
-import org.bukkit.Bukkit;
-import org.bukkit.Location;
-import org.bukkit.Sound;
-import org.bukkit.World;
+import org.bukkit.*;
```

Comment:
I'd like to avoid wildcard statements but I may let it slide cuz dis b a yolo code plogen

Reply:
IntellJ beeing a dum dum again, fixing it an other pr.